suites/suite_cufflinks_2_2_1/.shed.yml

```diff
@@ -0,0 +1,2 @@
+owner: devteam
```

Comment:
I think suites are mostly for the easiness of installation into Galaxy more than anything else. I agree it could have more information but devteam repo did not adopt the IUC advanced techniques yet.